tests/plugins/system-update/tst_updatemodel.cpp

```diff
@@ -392,6 +392,47 @@ private slots:
         QCOMPARE(update1->state(), Update::State::StateInstallFinished);
         QCOMPARE(update1->downloadId(), QString(""));
     }
+
+    //Test if the updated_at information is set after install only for click updates
+    void testClickUpdatedAtSetLate() {
+        auto update = createUpdate("id", 42);
+        update->setKind(Update::Kind::KindClick);
+        m_db->add(update);
+        m_model->setDownloaded(update->identifier(), update->revision());
+        auto update1 = m_db->get(update->identifier(), update->revision());
+        auto passed = update1->updatedAt() == QDateTime();
+        if (!passed)
+            QWARN("testClickUpdatedAtSetLate failed at early test!");
+        QVERIFY(passed);
```

Comment:
Can you please revert these superfluous logging changes to the tests, so that it just does the QCOMPARE() again?